### DISCUSSION TITLE
Display number of days since the habit was last performed

### DISCUSSION THREAD

**Discussion by mohanrajkumarplus** [iSoron/uhabits#657] [Feature Requests]
we can either add a number near the habit indicating, the number of days since we have done this habit

example - habit 1 ( 3 ) --> here 3 indicates, we did this habit 3 days back

on top of it we can provide a order by based on aging habits similar to "score", but this will be purely based on the days

**Comment by iSoron** [comment]
Thank you for the suggestion, @mohanrajkumarplus. I welcome proposals to display this data in the app's main screen, but I would like to see actual mockups of the proposed UI changes. I don't think showing this number inside parenthesis is a good idea.

**Reply by p-litwin** [reply]
Hello,
I'd also like to heave this feature. I prepared some simple visualisations how it could look like.
![Option 1](https://user-images.githubusercontent.com/27732762/104700899-24e74c80-5715-11eb-8d41-86e24f04cbc7.png)
![Option 2](https://user-images.githubusercontent.com/27732762/104700900-257fe300-5715-11eb-9503-95138b899105.png)

**Reply by naught101** [reply]
The tracked version of "days since last performed" would surely be "longest gap" or something, right? Mapped by month that could be a cool statistic.

**Reply by p-litwin** [reply]
I rather meant actual gap, but of course it could also show other statistics connected with gaps.
